index.html

```diff
@@ -217,6 +217,20 @@ <h2 class="fz20px">Skills</h2>
     </article>
   </main>
   <footer>
+    <article class="bg-footer border-tl">
+      <section class="footer_class p5vh">
+        <h2 class="title">Contact me</h2>
+        <p class="details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="flex flex-column" action="https://formspree.io/f/mlevqnqq" method="post">
+          <input type="text" id="name-input" name="name" placeholder="Your name" maxlength="30" required>
+          <input type="text" id="email" name="email" placeholder="Your email" required>
```

Comment:
- [ ] As this input is for the email, kindly change this type to `email`. In this way, you will have some auto [validations of email](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) made by the browser and it will save time writing some code. 💯